### PR TITLE
Update packages affected by yocto.

### PR DESCRIPTION
**Pull request by airstandley**

### CODE REVIEW

**Comment by airstandley** [comment]
On testing it looks like the networking package is failing due to `/etc/hostapd/hostapd.conf` not existing. Looking into it...

File: packages/buendia-networking/data/usr/bin/buendia-update-hosts [hunk 1]
```diff
@@ -29,7 +29,7 @@ unset LS_COLORS
 names=$(echo $(hostname) $(cat /usr/share/buendia/names.d/* | sed -e 's/#.*//'))
 
 # Add a line for each of the machine's IP addresses.
-for ip in $(ifconfig | grep -o 'inet addr:[0-9.]\+' | sed -e 's/.*://'); do
+for ip in $(ip a | grep -o 'inet [0-9.]\+' | sed -e 's/.*://'); do
```
Comment (airstandley) [inline packages/buendia-networking/data/usr/bin/buendia-update-hosts]: This doesn't quite work right. Needs a more exact pattern.

File: packages/buendia-networking/data/usr/share/buendia/config.d/networking [hunk 2]
```diff
@@ -46,87 +54,123 @@ fi
...
+    ignore_broadcast_ssid=0
+    rsn_pairwise=CCMP
+" > $hostapd_conf
+
+# Apply dnsmasq configuration
+unindex <<< "
+  interface=$WIFI_INTERFACE
+  listen-address=$NETWORKING_IP_ADDRESS
+  bind-interfaces
+" > /etc/dnsmasq.conf
```
Comment (airstandley) [inline packages/buendia-networking/data/usr/share/buendia/config.d/networking]: These are all placeholder configurations; need to get the correct configurations tests on a NUC and then update.

**Comment by schuyler** [comment]
note: this PR addresses #127

**Comment by schuyler** [comment]
closed in favor of #136

**Pull request closed without merging**